packages/components/form/src/form-validation-errors.ts

```diff
@@ -0,0 +1,110 @@
+import { localized, msg } from '@lit/localize';
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { InlineMessage } from '@sl-design-system/inline-message';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { type FormController } from './form-controller.js';
+import styles from './form-validation-errors.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-form-validation-errors': FormValidationErrors;
+  }
+}
+
+@localized()
+export class FormValidationErrors extends ScopedElementsMixin(LitElement) {
+  /** @internal */
+  static get scopedElements(): ScopedElementsMap {
+    return {
+      'sl-inline-message': InlineMessage
```

Comment:
Maybe `sl-icon` is missing?

<img width="465" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/8ae72042-6e07-4b93-a88e-5a1fe665b057">

Reply:
I have the inline-message in another PR, so i don't really want to check it until then.